### DISCUSSION TITLE
Can I have custom colours for the background/text?

### DISCUSSION THREAD

**Discussion by HubKing** [johnfactotum/foliate#1334] [Q&A]
In the colour settings, there are 9 presets, but none of them is what I want. Can I set custom colours?

![image](https://github.com/johnfactotum/foliate/assets/10900489/799c66e2-791a-4b94-affa-3ad4f53f48a8)

**Comment by johnfactotum** [comment]
See https://github.com/johnfactotum/foliate/blob/gtk4/docs/faq.md#how-to-use-custom-themes
